.gitignore

```diff
@@ -0,0 +1,6 @@
+*.py[cod]
+*$py.class
+configuration.py
+!configuration.py.example
+personal/
```

Comment:
Core/automation/jsr223/personal/**
Core/automation/lib/python/personal/**

We don't want to ignore the directory, just the files inside it. And it's theoretically possible that there could be another personal directory somewhere, so we could be specific about the path.

Reply:
Good point!